jams/core.py

```diff
@@ -1168,6 +1195,25 @@ def slice(self, start_time, end_time, strict=False):
 
         return sliced_ann
 
+    def to_interval_values(self):
+        '''Extract observation data in a `mir_eval`-friendly format.
+
+        Returns
+        -------
+        intervals : np.ndarray [shape=(n, 2), dtype=float]
+            Start- and end-times of all valued intervals
+
+            `intervals[i, :] = [time[i], time[i] + duration[i]]`
+
+        labels : list
+            List view of value field.
+        '''
+
+        times = timedelta_to_float(self.data.time.values)
+        duration = timedelta_to_float(self.data.duration.values)
+
+        return np.vstack([times, times + duration]).T, list(self.data.value)
```

Comment:
to concatenate them along the second axis; historical style holdover from the days before arbitary concat support.

but this is getting wiped in #149 also.